nix/overlay.nix

```
@@ -59,6 +59,23 @@ self: super: {
     cp $src/terraform-provider-hcloud $out/bin
   '';
 
+  nix = super.nix.overrideAttrs (drv: {
+    patches = drv.patches or [] ++ [
```

Nit, but I think this looks nicer/cleaner.

```suggestion
  nix = super.nix.overrideAttrs ({ patches ? [], ... }: {
    patches = patches ++ [
```